### PR TITLE
Version replacer readme

### PR DESCRIPTION
**Pull request by abhijoshi2k**
## Checks

I have...

- [x] read and understood the [Contributing Guidelines][contributing]
- [x] Updated any nessecary files such as the README.md and/or CHANGELOG.md.

## Type of Pull request

- [x] **Minor Change**  
  This Pull request doesn't break existing configuration.
- [ ] **Major Change**  
  This Pull request will break existing configuration.
- [ ] **Bug fix**  
  This Pull request will fix a (critical) bug.
- [ ] **Documentation**  
  This Pull request only changes documentation (README.md, CHANGELOG.md, etc.)
- [ ] **Other:** `__________` 

## Description

Added workflow to automate version updation in `README.md` workflow structure.
Changes in readme to be done through `TEMPLATE.md` now.

Closes #

[contributing]: https://github.com/Readme-Workflows/.github/blob/main/.github/CONTRIBUTING.md

### CODE REVIEW

**Comment by PuneetGopinath** [comment]
Wow!
This will help us a lot, can we run this one more time to replace TBA with latest release date? Is that possible using eval?

File: .github/customReplacer.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[
+  {
+    "search": "{version}",
+    "replace": "'v'+require('../package.json').version",
```
Comment (PuneetGopinath) [inline .github/customReplacer.json]:
```suggestion
    "replace": "'v' + require('../package.json').version",
```
Small code style

**Comment by abhijoshi2k** [comment]
> Wow!
> This will help us a lot, can we run this one more time to replace TBA with latest release date? Is that possible using eval?

Yes

**Comment by PuneetGopinath** [comment]
> > Wow!
> > This will help us a lot, can we run this one more time to replace TBA with latest release date? Is that possible using eval?
> 
> Yes

Cool, can you do that too! This is what @Andre601 asked, he said `Can we make a workflow to replace release date everytime we release in Changelog` days ago

**Comment by abhijoshi2k** [comment]
@PuneetGopinath I'll have to run that workflow before release. There's no way to run workflow and then release!

**Comment by PuneetGopinath** [comment]
> @PuneetGopinath I'll have to run that workflow before release. There's no way to run workflow and then release!

You can run after release

**Comment by abhijoshi2k** [comment]
> > @PuneetGopinath I'll have to run that workflow before release. There's no way to run workflow and then release!
> 
> You can run after release

Okay. Can you tell me exact file path where I have to replace data? And the placeholder for it. Also, what format of date is expected? And what timezone is expected?

**Comment by PuneetGopinath** [comment]
> Okay. Can you tell me exact file path where I have to replace data? And the placeholder for it. Also, what format of date is expected? And what timezone is expected?

Placeholder: `{TBA}`
Format: `DD-MM-YYYY`
Timezone: UTC
Replace To: `CHANGELOG.md`

**Comment by abhijoshi2k** [comment]
@PuneetGopinath Please check.

**Comment by PuneetGopinath** [comment]
> @PuneetGopinath Please check.

Looks okay, have to test
Before that why do you need to create another workflow, in the same workflow you created you can add another job or even another step.

**Comment by abhijoshi2k** [comment]
I'll add a step

**Comment by abhijoshi2k** [comment]
Done

**Review by Andre601 — APPROVED** [review]

**Review by PuneetGopinath — APPROVED** [review]

**Comment by PuneetGopinath** [comment]
@abhijoshi2k Please resolve conflicts

**Pull request merged**